### DISCUSSION TITLE
archives/bj-pfd/home/

### DISCUSSION THREAD

**Discussion by giscus** [songtianlun/songtianlun.github.io#12] [Announcements]
# archives/bj-pfd/home/

Bullet Journal Personal Finance Dashboard (BJ-PFD) ，一个 Notion 子弹笔记（财务向）辅助看板。

https://www.frytea.com/archives/bj-pfd/home/

**Comment by fupre** [comment]
您好，这些文档都404了，BJ-PFD还可以使用吗,看了之前的文章对您这个项目很喜欢，也想使用下

**Reply by songtianlun** [reply]
https://www.frytea.com/tags/bj-pfd2/
